src/haproxy.py

```diff
@@ -106,5 +106,13 @@ def _render_haproxy_config(self, config: CharmConfig) -> None:
         self._render_file(HAPROXY_CONFIG, rendered, 0o644)
 
     def _reload_haproxy_service(self) -> None:
-        """Restart the haporxy service."""
-        systemd.service_reload(HAPROXY_SERVICE)
+        """Reload the haporxy service.
```

Comment:
Nit pick:
`s/haporxy/haproxy`

Reply:
Thanks! will update that in the next PR.